server/stat.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2020 Google LLC
```

Comment:
I assume the logic here mimics the existing logic except for populating proto?

Reply:
Yes, they have the same logci.